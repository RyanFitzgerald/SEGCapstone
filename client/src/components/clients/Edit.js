import React from 'react';
import { Redirect } from 'react-router-dom';
import * as api from '../../api';

import Loading from '../Loading';

class Edit extends React.Component {
  constructor() {
    super();

    // Bind functions
    this.handleSubmit = this.handleSubmit.bind(this);
    this.getClient = this.getClient.bind(this);
    this.updateClient = this.updateClient.bind(this);
    this.handlePhone = this.handlePhone.bind(this);

    // Set state
    this.state = {
      redirect: false,
      client: null
    }
  }

  componentDidMount() {
    // Update tab
    this.props.setActiveSubtab(0);

    // Get client
    this.getClient(this.props.location.match.params.id);
  }

  handleSubmit(e) {
    // Stop form submission
    e.preventDefault();

    // Get form data
    const client = {
      name: this.name.value,
      telephone: this.telephone.value,
      email: this.email.value,
      street: this.street.value,
      postalCode: this.postalCode.value,
      city: this.city.value
    };

    // Call api
    this.updateClient(client);
  }

  getClient(id) {
    api.getClient(id).then(client => {
      this.setState({ client }, () => {
        // Set title
        document.title = `Edit ${this.state.client.name} | Renovaction`;
      })
    });
  }

  updateClient(client) {
    api.updateClient(client, this.props.location.match.params.id).then(resp => {
      // Append id
      client._id = resp;

      // Update parent state
      this.props.updateClients(client);

      // Redirect
      this.setState({
        redirect: `/clients/${resp}`
      });
    });
  }

  handlePhone(e) {
    e.target.value = e.target.value.replace(/^(\d{3})(\d{3})(\d)+$/, '$1-$2-$3');
  }

  render() {
    if (this.state.redirect) {
      return (
        <Redirect to={this.state.redirect}/>
      );
    }

    if (this.state.client) {
      return (
        <div className="content">
          <div className="row">
            <div className="column">
              <h2 className="card-title">Add a New Client</h2>
              <div className="card">
                <form onSubmit={this.handleSubmit}>
                  <div className="row form-section">
                    <div className="md-4 column form-section__title no-left">
                      <h3>Basic Information</h3>
                      <p>
                        Enter all the basic information about this client to help you identify them later
                      </p>
                    </div>
                    <div className="md-8 column no-right">
                      <label className="form-label" htmlFor="name"> Name <span className="form-required">*</span></label>
                      <input ref={input => this.name = input} name="name" className="form-text form-text--full" type="text" defaultValue={this.state.client.name} required />
                      <label className="form-label" htmlFor="salesman">Sold by <span className="form-required">*</span></label>
                      <span className="form-select">
                        <select ref={input => this.soldBy = input} name="salesman" required>
                          <option>John Doe</option>
                        </select>
                      </span>
                    </div>
                  </div>
                  <div className="row form-section">
                      <div className="md-4 column form-section__title no-left">
                          <h3>Contact Information</h3>
                          <p>
                              Enter all the contact information about this client
                          </p>
                      </div>
                      <div className="md-8 column no-right">
                          <label className="form-label" htmlFor="email">Email <span className="form-required">*</span></label>
                          <input ref={input => this.email = input} name="email" className="form-text form-text--full" type="email" defaultValue={this.state.client.email} required/>
                          <label className="form-label" htmlFor="phone">Phone Number <span className="form-required">*</span></label>
                          <input ref={input => this.telephone = input} name="phone" className="form-text form-text--full" type="text" onKeyUp={this.handlePhone} maxLength="12" defaultValue={this.state.client.telephone} required/>
                      </div>
                  </div>
                  <div className="row form-section no-border">
                    <div className="md-4 column form-section__title no-left">
                        <h3>Client Location</h3>
                        <p>
                            Enter all the information associated with the location of this client
                        </p>
                    </div>
                    <div className="md-8 column no-right">
                      <label className="form-label" htmlFor="street">Street <span className="form-required">*</span></label>
                      <input ref={input => this.street = input} name="street" className="form-text form-text--full" type="text" defaultValue={this.state.client.street} required/>
                      <label className="form-label" htmlFor="postal-code">Postal Code <span className="form-required">*</span></label>
                      <input ref={input => this.postalCode = input} name="postal-code" className="form-text form-text--full capitalize" type="text" maxLength="6" defaultValue={this.state.client.postalCode} required/>
                      <label className="form-label" htmlFor="city">City <span className="form-required">*</span></label>
                      <span className="form-select">
                        <select ref={input => this.city = input} name="city" defaultValue={this.state.client.city} required>
                          <option value="Ottawa">Ottawa</option>
                        </select>
                      </span>
                    </div>
                  </div>
                  <div className="text-center">
                      <input type="submit" className="btn btn--primary btn--large" value="Update Client"/>
                  </div>
                </form>
              </div>
            </div>
          </div>
        </div>
      );
    }

    return (
      <Loading/>
    );
  }
}

export default Edit;