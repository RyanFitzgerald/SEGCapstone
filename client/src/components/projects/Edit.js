import React from 'react';
import { Redirect } from 'react-router-dom';
import * as api from '../../api';
import moment from 'moment';

import TagSelectorWrapper from '../TagSelectorWrapper';
import PikadayWrapper from '../PikadayWrapper';

import Loading from '../Loading';

class Edit extends React.Component {
  constructor() {
    super();

    // Bind functions
    this.handleSubmit = this.handleSubmit.bind(this);
    this.updateProject = this.updateProject.bind(this);
    this.getCents = this.getCents.bind(this);
    this.getDollars = this.getDollars.bind(this);
    this.contentLoaded = this.contentLoaded.bind(this);
    this.getProject = this.getProject.bind(this);

    // Set state
    this.state = {
      redirect: false,
      clients: false,
      types: false,
      project: false,
      formError: false
    }
  }

  componentDidMount() {
    // Set title
    document.title = 'Edit Project | Renovaction';

    // Update tab
    this.props.setActiveSubtab(0);

    // Get project
    this.getProject(this.props.location.match.params.id);
  }

  componentWillReceiveProps(nextProps) {
    if (nextProps.clients !== null && nextProps.clients !== this.state.clients) {
      this.setState({ clients: nextProps.clients });
    }

    if (nextProps.types !== null && nextProps.types !== this.state.types) {
      this.setState({ types: nextProps.types });
    }
  }

  handleSubmit(e) {
    e.preventDefault();

    let types = [];
    for (let i = 0; i < this.projectType.selectField.options.length; i++) {
      if (this.projectType.selectField.options[i].selected) {
        types.push(this.projectType.selectField.options[i].value);
      }
    }

    if (types.length < 1) {
      this.setState({
        formError: 'You must select a project type.'
      });
      return;
    }

    // Get form data
    const project = {
      fileNumber: this.fileNumber.value,
      houseNumber: this.houseNumber.value,
      street: this.street.value,
      city: this.city.value,
      postalCode: this.postalCode.value,
      soldDate: this.soldDate.field.value,
      startDate: this.startDate.field.value,
      endDate: this.endDate.field.value,
      cashinDate: this.cashinDate.field.value,
      labourCost: this.getCents(this.labourCost.value),
      materialsCost: this.getCents(this.materialsCost.value),
      contractCost: this.getCents(this.contractCost.value),
      status: this.status.value,
      type: types,
      client: this.client.value,
      access_token: JSON.parse(sessionStorage.getItem('user')).access_token
    };

    // Call api
    this.updateProject(project);
  }

  getProject(id) {
    const query = {
      id,
      access_token: JSON.parse(sessionStorage.getItem('user')).access_token
    }

    api.getProject(query).then(project => {
      this.setState({ project }, () => {
        // Set title
        document.title = `Edit ${this.state.project.name} | Renovaction`;
      })
    });
  }

  updateProject(project) {
    api.updateProject(project, this.props.location.match.params.id).then(resp => {
      if (resp.status === 500) {
        this.setState({
          formError: 'There was an error when submitting the form, please try again.'
        })
        return;
      }

      // Update parent state
      this.props.getProjects({search: false});

      // Redirect
      this.setState({
        redirect: {
          location: `/projects/${resp._id}`,
          message: 'Successfully updated project!',
          type: 'success'
        }
      });
    });
  }

  getCents(dollars) {
    return Math.round((Math.round(dollars * 100) / 100)*100);
  }

  getDollars(cents) {
    const dollars = (cents/100).toFixed(2);
    let dollarString = dollars.toString().split('.');

    if (dollarString[0].length >= 4) {
      dollarString[0] = dollarString[0].replace(/(\d)(?=(\d{3})+$)/g, '$1,');
    }

    return dollarString.join('.');
  }

  contentLoaded() {
    return this.state.clients && this.state.types && this.state.project;
  }

  render() {

    if (this.state.redirect) {
      this.props.addNotification(this.state.redirect.message, this.state.redirect.type);
      return (
        <Redirect to={this.state.redirect.location} />
      );
    }

    if (!this.contentLoaded()) {
      return (
        <Loading/>
      );
    }

    return (
      <div className="content">
        <div className="row">
          <div className="column">
            <h2 className="card-title">Add a New Project</h2>
            <div className="card">
              {this.props.renderError(this.state.formError)}
              <form onSubmit={this.handleSubmit}>
                <div className="row form-section">
                  <div className="md-4 column form-section__title no-left">
                    <h3>Basic Information</h3>
                    <p>
                      Enter all the basic information about this project to help you identify it later
                    </p>
                  </div>
                  <div className="md-8 column no-right">
                    <label className="form-label" htmlFor="fileNumber">Project File Number <span className="form-required">*</span></label>
                    <input name="fileNumber" ref={input => this.fileNumber = input} className="form-text form-text--full" type="text" placeholder="E.g. 2017-435" defaultValue={this.state.project.fileNumber} required/>
                    <label className="form-label" htmlFor="client">Project Client <span className="form-required">*</span></label>
                    <span className="form-select">
                      <select name="client" ref={input => this.client = input} defaultValue={this.state.project.client.id} required>
                      {this.state.clients.map((client, key) => {
                        return <option key={key} value={client._id}>{client.firstName} {client.lastName}</option>;
                      })}
                      </select>
                    </span>
                    <label className="form-label" htmlFor="client">Project Status <span className="form-required">*</span></label>
                    <span className="form-select">
                      <select name="client" ref={input => this.status = input} defaultValue={this.state.project.status} required>
                        <option value="Not Started">Not Started</option>
                        <option value="In Progress">In Progress</option>
                        <option value="Complete">Complete</option>
                      </select>
                    </span>
                  </div>
                </div>
                <div className="row form-section">
                  <div className="md-4 column form-section__title no-left">
                    <h3>Project Location</h3>
                    <p>
                      Enter all the information associated with the location of this project
                    </p>
                  </div>
                  <div className="md-8 column no-right">
                    <label className="form-label" htmlFor="houseNumber">House Number <span className="form-required">*</span></label>
                    <input name="houseNumber" ref={input => this.houseNumber = input} className="form-text form-text--full" type="text" defaultValue={this.state.project.houseNumber} required/>
                    <label className="form-label" htmlFor="street">Street <span className="form-required">*</span></label>
                    <input name="street" ref={input => this.street = input} className="form-text form-text--full" type="text" defaultValue={this.state.project.street} required/>
                    <label className="form-label" htmlFor="city">City <span className="form-required">*</span></label>
                    <input name="city" ref={input => this.city = input} className="form-text form-text--full" type="text" defaultValue={this.state.project.city} required/>
                    <label className="form-label" htmlFor="postal-code">Postal Code <span className="form-required">*</span></label>
                    <input name="postal-code" ref={input => this.postalCode = input} className="form-text form-text--full capitalize" maxLength="6" type="text" defaultValue={this.state.project.postalCode} required/>
                  </div>
                </div>
                <div className="row form-section">
                  <div className="md-4 column form-section__title no-left">
                    <h3>Project Type</h3>
                    <p>
                      Enter all information about this project's type(s)
                    </p>
                  </div>
                  <div className="md-8 column no-right">
                    <label className="form-label" htmlFor="type">Type(s) <span className="form-required">*</span></label>
                    <TagSelectorWrapper name="type" id="project-type" ref={input => this.projectType = input} defaultValue={this.state.project.type}>
                      {this.state.types.map((type, key) => {
                        return <option key={key} value={type._id}>{type.name}</option>;
                      })}
                    </TagSelectorWrapper>
                  </div>
                </div>
                <div className="row form-section">
                  <div className="md-4 column form-section__title no-left">
                    <h3>Project Dates</h3>
                    <p>
                      Enter all known date information associated with this project (Format: January 1, 2017)
                    </p>
                  </div>
                  <div className="md-8 column no-right">
                    <label className="form-label" htmlFor="sold-date">Sold Date <span className="form-required">*</span></label>
                    <PikadayWrapper name="sold-date" ref={input => this.soldDate = input} id="sold-date" defaultValue={moment(this.state.project.soldDate).format('MMMM DD, YY')} required="true"/>
                    <label className="form-label" htmlFor="cashin-date">Cashin Date</label>
                    <PikadayWrapper name="cashin-date" ref={input => this.cashinDate = input} id="cashin-date" defaultValue={(this.state.project.cashinDate) ? moment(this.state.project.cashinDate).format('MMMM DD, YY') : ''} />
                    <label className="form-label" htmlFor="start-date">Project Start Date</label> 
                    <PikadayWrapper name="start-date" ref={input => this.startDate = input} id="start-date" defaultValue={(this.state.project.startDate) ? moment(this.state.project.startDate).format('MMMM DD, YY') : ''} />
                    <label className="form-label" htmlFor="end-date">Project End Date</label>
                    <PikadayWrapper name="end-date" ref={input => this.endDate = input} id="end-date" defaultValue={(this.state.project.endDate) ? moment(this.state.project.endDate).format('MMMM DD, YY') : ''} />
                  </div>
                </div>
                <div className="row form-section no-border">
                  <div className="md-4 column form-section__title no-left">
                    <h3>Project Costs</h3>
                    <p>
                      Enter all known cost information associated with this project
                    </p>
                  </div>
                  <div className="md-8 column no-right">
                    <label className="form-label" htmlFor="labour-cost">Labour Cost</label>
                    <input name="labour-cost" ref={input => this.labourCost = input} defaultValue={this.getDollars(this.state.project.labourCost).replace(/,/g, '')} className="form-text form-text--full" type="number" step="0.01" />
                    <label className="form-label" htmlFor="materials-cost">Materials Cost</label>
                    <input name="materials-cost" ref={input => this.materialsCost = input} defaultValue={this.getDollars(this.state.project.materialsCost).replace(/,/g, '')} className="form-text form-text--full" type="number" step="0.01" />
                    <label className="form-label" htmlFor="actual-cost">Contract Cost</label>
                    <input name="actual-cost" ref={input => this.contractCost = input} defaultValue={this.getDollars(this.state.project.contractCost).replace(/,/g, '')} className="form-text form-text--full" type="number" step="0.01" />
                  </div>
                </div>
                <div className="text-center">
                  <input type="submit" className="btn btn--primary btn--large" value="Update Project" />
                </div>
              </form>
            </div>
          </div>
        </div>
      </div>
    );
    
  }
}

export default Edit;