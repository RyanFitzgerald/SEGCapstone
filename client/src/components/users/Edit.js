import React from 'react';
import { Redirect } from 'react-router-dom';
import * as api from '../../api';

import Loading from '../Loading';

class Edit extends React.Component {
  constructor() {
    super();

    // Bind functions
    this.handleSubmit = this.handleSubmit.bind(this);
    this.getUser = this.getUser.bind(this);
    this.updateUser = this.updateUser.bind(this);

    // Set state
    this.state = {
      redirect: false,
      user: null,
      formError: false
    }
  }

  componentDidMount() {
    // Update tab
    this.props.setActiveSubtab(0);

    // Get user
    this.getUser(this.props.location.match.params.id);
  }

  handleSubmit(e) {
    // Stop form submission
    e.preventDefault();

    // Get form data
    const user = {
      name: this.name.value,
      email: this.email.value,
      password: this.password.value,
      access_token: JSON.parse(sessionStorage.getItem('user')).access_token
    };

    // Call api
    this.updateUser(user);
  }

  getUser(id) {
    const query = {
      id,
      access_token: JSON.parse(sessionStorage.getItem('user')).access_token
    }

    api.getUser(query).then(user => {
      this.setState({ user }, () => {
        // Set title
        document.title = `Edit ${this.state.user.name} | Renovaction`;
      })
    });
  }

  updateUser(user) {
    api.updateUser(user, this.props.location.match.params.id).then(resp => {
      if (resp.status === 500) {
        this.setState({
          formError: 'There was an error when submitting the form, please try again.'
        })
        return;
      }

      // Append id
      user._id = resp._id;

      // Update parent state
      this.props.updateUsers(resp);

      // Redirect
      this.setState({
        redirect: {
          location: `/settings/users/${resp._id}`,
          message: 'Successfully updated user!',
          type: 'success'
        }
      });
    });
  }

  render() {
    if (this.state.redirect) {
      this.props.addNotification(this.state.redirect.message, this.state.redirect.type);
      return (
        <Redirect to={this.state.redirect.location}/>
      );
    }

    if (this.state.user) {
      return (
        <div className="content">
          <div className="row">
            <div className="column">
              <h2 className="card-title">Edit User</h2>
              <div className="card">
                {this.props.renderError(this.state.formError)}
                <form onSubmit={this.handleSubmit}>
                  <div className="row">
                    <div className="md-4 column form-section__title no-left">
                      <h3>Basic Information</h3>
                      <p>
                        Enter all the basic information about this user
                      </p>
                    </div>
                    <div className="md-8 column no-right">
                      <label className="form-label" htmlFor="name"> Name <span className="form-required">*</span></label>
                      <input ref={input => this.name = input} name="name" className="form-text form-text--full" type="text" defaultValue={this.state.user.name} required />
                      <label className="form-label" htmlFor="email">Email <span className="form-required">*</span></label>
                      <input ref={input => this.email = input} name="email" className="form-text form-text--full" type="email" defaultValue={this.state.user.email} required/>
                      <label className="form-label" htmlFor="password">Password <span className="form-required">*</span></label>
                      <input ref={input => this.password = input} name="password" className="form-text form-text--full" type="password" placeholder="(keep same password)" />
                    </div>
                  </div>
                  <div className="text-center">
                      <input type="submit" className="btn btn--primary btn--large" value="Update User"/>
                  </div>
                </form>
              </div>
            </div>
          </div>
        </div>
      );
    }

    return (
      <Loading/>
    );
  }
}

export default Edit;