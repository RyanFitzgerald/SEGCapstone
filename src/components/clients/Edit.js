import React, { Component, PropTypes } from 'react';
import * as api from '../../api';
import Loading from '../Loading';

class Edit extends Component {
    constructor(props) {
        super(props);

        this.state = {
            client: null
        };
    }

    componentDidMount() {
        // Update page title
        document.title = `Edit Client ${this.props.match.params.clientID}`;

        // Update Active Tab (zero to hide)
        this.props.setHeaderTab(0);

        // Get client
        this.getClient(this.props.match.params.clientID);
    }

    getClient = clientID => {
        api.getClientByID(clientID).then(client => {
            this.setState({
                client
            });
        });
    }

    handleSubmit = e => {
        e.preventDefault();
        let newClient = {
            clientID: this.refs.clientID.value,
            clientName: this.refs.name.value,
            email: this.refs.email.value,
            telephone: this.refs.phone.value,
            street: this.refs.street.value,
            postalCode: this.refs.postalCode.value,
            cityID: this.refs.city.value
        };
        this.props.editClient(newClient);
    };

    handlePhone = e => {
        e.target.value = e.target.value.replace(/^(\d{3})(\d{3})(\d)+$/, '($1) $2-$3');
    };

    currentContent() {
        if (this.state.client) {
            return (
                <div id="dashboard-other">
                    <div className="row">
                        <div className="small-12 columns">
                            <h2>Update Client</h2>
                            <form onSubmit={this.handleSubmit}>
                                <input type="hidden" ref="clientID" name="client_id" value={this.state.client.client_id} />
                                <div id="client-add" className="dashboard-block">
                                    <div className="row">
                                        <div className="small-12 columns">
                                            <label htmlFor="client-first-name">Name <span className="required">*</span></label>
                                            <input type="text" ref="name" name="name" className="client-text-form" id="client-first-name" defaultValue={this.state.client.name} required />
                                        </div>
                                    </div>
                                    <div className="row">
                                        <div className="small-12 medium-6 columns">
                                            <label htmlFor="client-email">Email</label>
                                            <input type="email" ref="email" name="email" className="client-text-form" id="client-email" defaultValue={this.state.client.email} />
                                        </div>
                                        <div className="small-12 medium-6 columns">
                                            <label htmlFor="client-phone">Phone Number <span className="required">*</span></label>
                                            <input type="text" ref="phone" name="phone" className="client-text-form" id="client-phone" maxLength="14" onKeyUp={this.handlePhone} defaultValue={this.state.client.telephone} required />
                                        </div>
                                    </div>
                                    <div className="row">
                                        <div className="small-12 medium-4 columns">
                                            <label htmlFor="client-street">Street <span className="required">*</span></label>
                                            <input type="text" ref="street" name="street" className="client-text-form" id="client-street" defaultValue={this.state.client.street} required />
                                        </div>
                                        <div className="small-12 medium-4 columns">
                                            <label htmlFor="client-postal-code">Postal Code <span className="required">*</span></label>
                                            <input type="text" ref="postalCode" name="postalCode" className="client-text-form postal-code-field" id="client-postal-code" defaultValue={this.state.client.postal_code} maxLength="6" required />
                                        </div>
                                        <div className="small-12 medium-4 columns">
                                            <label htmlFor="client-city">City <span className="required">*</span></label>
                                            <select name="city" ref="city" className="client-select-form" id="client-city" defaultValue={this.state.client.city_id} required>
                                                {this.props.cities.map(city => {
                                                    return <option key={city.city_id} value={city.city_id}>{city.city}</option>;
                                                })}
                                            </select>
                                        </div>
                                    </div>
                                    <p className="required-info">Fields marked with * are required.</p>
                                    <input type="submit" name="submit" className="btn" id="client-submit" value="Update Client" />
                                </div>
                                {/* <-- End #client-add -->*/}
                            </form>
                        </div>
                    </div>
                </div>
            );
        }

        return (
            <Loading />
        );
    }

    render() {
        return (
            <div>
                {this.currentContent()}
            </div>
        );
    }
}

Edit.propTypes = {
    match: PropTypes.any,
    setHeaderTab: PropTypes.func.isRequired,
    cities: PropTypes.array.isRequired,
    editClient: PropTypes.func.isRequired
};

export default Edit;
