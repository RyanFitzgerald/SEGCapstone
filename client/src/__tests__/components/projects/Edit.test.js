import React from 'react';
import Edit from '../../../Components/projects/Edit';

describe('<Edit />', () => {
  let location;

  beforeAll(() => {
    // Add necessary info to user
    const user = {
      role: {
        level: 1
      }
    };
    sessionStorage.setItem('user', JSON.stringify(user));

    // Create location prop with needed data
    location = {
      match: {
        params: {
          id: 1
        }
      }
    };
  });

  it('should render correctly', () => {   
    // Return empty objects on requests
    mock.onAny().reply(200, {});

    // Create mock functions / spies
    const setActiveSubtabMock = jest.fn();
    const renderErrorMock = jest.fn();
    const getProjectSpy = jest.spyOn(Edit.prototype, 'getProject');
    
    // Shallow render it
    const wrapper = shallow(
      <Edit setActiveSubtab={setActiveSubtabMock} renderError={renderErrorMock} location={location}/>
    );

    // Compare to old snapshot
    expect(shallowToJson(wrapper)).toMatchSnapshot();

    // Check set active subtab is called and with correct value
    expect(setActiveSubtabMock).toHaveBeenCalledTimes(1);
    expect(setActiveSubtabMock.mock.calls[0][0]).toBe(0);

    // Check if component mounting methods were called
    expect(getProjectSpy).toHaveBeenCalled();
  });
});