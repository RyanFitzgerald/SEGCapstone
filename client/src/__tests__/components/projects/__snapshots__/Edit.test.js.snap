// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Edit /> should render correctly 1`] = `<Loading />`;
